packages/components/src/flex/flex.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { forwardRef } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { Flex as BaseFlex } from './styles/flex-styles';
+
+function Flex(
```

Comment:
A layout primitive! Very handy for many common UI cases ✌️
It was used within `BoxControl`

Reply:
Do you think we need a README for this?

Reply:
It's a bit unclear what this gives me over CSS flexbox? Why would I use this instead?